openedx/core/djangoapps/programs/utils.py

```diff
@@ -22,6 +21,7 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.core.djangoapps.programs.models import ProgramsApiConfig
 from openedx.core.lib.edx_api_utils import get_edx_api_data
+
```

Comment:
Minor, but I noticed it wasn't fixed: remove this newline. These imports are all first-party (they belong to us) and should be grouped together. The convention is three blocks: one for standard library imports, one for third-party imports, and one for any first-party imports.